CONTRIBUTING.md

```diff
@@ -8,12 +8,12 @@ Follow these steps to make a contribution to any of CF open source repositories:
    [individuals](http://cloudfoundry.org/pdfs/CFF_Individual_CLA.pdf) or
    [corporations](http://cloudfoundry.org/pdfs/CFF_Corporate_CLA.pdf).
 
-1. Set your name and email (these should match the information on your submitted CLA)
+2. Set your name and email (these should match the information on your submitted CLA)
 
         git config --global user.name "Firstname Lastname"
         git config --global user.email "your_email@example.com"
 
-1. If your company has signed a Corporate CLA, but sure to make the membership in your company's github organization public
+3. If your company has signed a Corporate CLA, but sure to make the membership in your company's github organization public
```

Comment:
Just using `1.` is actually a feature of markdown. Second example here: https://www.markdownguide.org/basic-syntax/#ordered-lists